### PR TITLE
Python 3.6 ubuntu 16.04

### PR DESCRIPTION
**Pull request by jburel**
* Indicate the version of Python installed
* Add option to install Python 3.6 on ubuntu 16.04 from source
* The installation is tested via travis

see https://github.com/ome/omeroweb-install/issues/61

### CODE REVIEW

File: ansible/templates/common/virtualenv_py36.sh.j2 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{% if doc %}
+If you are using Python 3.6. Run the following command instead::
```
Comment (mtbc) [inline ansible/templates/common/virtualenv_py36.sh.j2:2]:
```suggestion
If you are using Python 3.6 then run the following command instead::
```

File: ansible/templates/ubuntu1604/deps.sh.j2 [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
-{% endif %}
-
 apt-get update
-
 apt-get -y install unzip
+apt-get -y install nginx
+
+if [ "${PYTHONVER:-}" = "3.6" ]; then
+    apt-get install -y software-properties-common
+    add-apt-repository ppa:deadsnakes/ppa
```
Comment (joshmoore) [inline ansible/templates/ubuntu1604/deps.sh.j2:25]: Was the intent to remove PPA usage everywhere?

Comment (jburel) [reply ansible/templates/ubuntu1604/deps.sh.j2:25]: oops this one was forgotten
It should have been removed

**Comment by dominikl** [comment]
Looks good to me 👍 (But bear in mind my limited ansible expertise)

**Pull request merged**